ocis-phoenix/docs/getting-started.md

```diff
@@ -0,0 +1,339 @@
+
+PHOENIX_OIDC_METADATA_URL
+: OpenID Connect metadata URL, defaults to `http://localhost:9130/.well-known/openid-configuration`
+
+PHOENIX_OIDC_AUTHORITY
+: OpenID Connect authority, defaults to `http://localhost:9130`
+
+PHOENIX_OIDC_CLIENT_ID
+: OpenID Connect client ID, defaults to `phoenix`
+
+PHOENIX_OIDC_RESPONSE_TYPE
+: OpenID Connect response type, defaults to `code`
+
+PHOENIX_OIDC_SCOPE
+: OpenID Connect scope, defaults to `openid profile email`
+
+In case you want to render any additional properties in the config.json you can provide a custom config.json using eg. `PHOENIX_WEB_CONFIG=/path/to/config.json ocis-phoenix server`
+
+#### Health
+
+PHOENIX_DEBUG_ADDR
+: Address to debug endpoint, defaults to `0.0.0.0:9104`
+
+### Commandline flags
+
+If you prefer to configure the service with commandline flags you can see the available variables below.
+
+#### Global
+
+--config-file
+: Path to config file, empty default value
+
+--log-level
+: Set logging level, defaults to `info`
+
+--log-color
+: Enable colored logging, defaults to `true`
+
+--log-pretty
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+--tracing-enabled
+: Enable sending traces, defaults to `false`
+
+--tracing-type
+: Tracing backend type, defaults to `jaeger`
+
+--tracing-endpoint
+: Endpoint for the agent, empty default value
+
+--tracing-collector
+: Endpoint for the collector, empty default value
+
+--tracing-service
+: Service name for tracing, defaults to `phoenix`
+
+--debug-addr
+: Address to bind debug server, defaults to `0.0.0.0:9104`
+
+--debug-token
+: Token to grant metrics access, empty default value
+
+--debug-pprof
+: Enable pprof debugging, defaults to `false`
+
+--debug-zpages
+: Enable zpages debugging, defaults to `false`
+
+--http-addr
+: Address to bind http server, defaults to `0.0.0.0:9100`
+
+--http-root
+: Root path of http server, defaults to `/`
+
+--asset-path
+: Path to custom assets, empty default value
+
+--web-config
+: Path to phoenix config, empty default value
+
+--web-config-server
+: Server URL, defaults to `http://localhost:9135`
+
+--web-config-theme
+: Theme, defaults to `owncloud`
+
+--web-config-version
+: Version, defaults to `0.1.0`
+
+--web-config-app
+: Provide multiple apps, defaults to `""`. Usage: `phoenix --web-config-app files --web-config-app pdf-viewer`
+
+--oidc-metadata-url
+: OpenID Connect metadata URL, defaults to `http://localhost:9130/.well-known/openid-configuration`
+
+--oidc-authority
+: OpenID Connect authority, defaults to `http://localhost:9130`
+
+--oidc-client-id
+: OpenID Connect client ID, defaults to `phoenix`
+
+--oidc-response-type
+: OpenID Connect response type, defaults to `code`
+
+--oidc-scope
+: OpenID Connect scope, defaults to `openid profile email`
+
+In case you want to render any additional properties in the config.json you can provide a custom config.json using  eg. `ocis-phoenix server --web-config=/path/to/config.json`
+
+#### Health
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (119:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)